onnxruntime/test/providers/tensorrt/tensorrt_basic_test.cc

```diff
@@ -26,21 +34,66 @@ void VerifyOutputs(const std::vector<OrtValue>& fetches, const std::vector<int64
   ASSERT_EQ(expected_values, found);
 }
 
-TEST(TensorrtExecutionProviderTest, EngineCachingTest) {
-  ScopedEnvironmentVariables scoped_env_vars{EnvVarMap{
-      {"ORT_TENSORRT_ENGINE_CACHE_ENABLE", {"1"}},
-  }};
-  onnxruntime::Model model("enginecachingtest", false, DefaultLoggingManager().DefaultLogger());
+bool IsTensorRTCacheExisted(std::string path, std::string file_extension) {
```

Comment:
can any of these be consolidated with existing methods and 
and any utility methods related to trt ep all moved to the tensorrt_execution_provider_utils.h ?

Reply:
We have only one existing GetCachePath() and that only works with known cache name. It's hard for these functions associated with cache to be consolidated with this existing one. 
I moved several general utility methods to tensorrt_execution_provider_utils.h